src/pudl_rmi/deprish.py

```diff
@@ -30,14 +30,14 @@
 logger = logging.getLogger(__name__)
 
 
-INT_IDS = ['utility_id_ferc1', 'utility_id_pudl',
+INT_IDS = ['utility_id_ferc1', 'utility_id_pudl', 'plant_id_eia',
            'plant_id_pudl', 'report_year']
```

Comment:
Is this supposed to have both `plant_id_eia` and `plant_id_pudl`?

Reply:
good catch! I had both of these in here for a bit when I had both ID's in the raw depreciation file, but I believe I can remove this now bc I've removed the pudl id.